_codux/board-wrappers/component-wrapper.tsx

```diff
@@ -1,6 +1,6 @@
 import { createRemixStub } from '@remix-run/testing';
 import { PropsWithChildren } from 'react';
-import { EcomAPIContextProvider } from '~/api/ecom-api-context-provider';
+import { EcomAPIContextProvider } from 'lib/ecom';
```

Comment:
Oh, I didn't realize imports are going to look like this, `lib` looks like a name of an NPM package. For some reason I though it's going to be `~/lib/ecom`.

Maybe we should do this?
`~/src/components`
`~/lib/components`

Reply:
Even without `lib`, `~/src/router` is more intuitive for me than `~/router`

Reply:
Updated all imports to the pattern
`~/src/components`
`~/lib/components`